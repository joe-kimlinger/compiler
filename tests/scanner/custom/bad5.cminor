// this is a comment \n and it doesnt end there
/* comment @ *
