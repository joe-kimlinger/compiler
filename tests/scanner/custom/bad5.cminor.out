COMMENT
SLASH
STAR
IDENTIFIER comment
